packages/shared/lib/core/app/constants/default-app-settings.constant.ts

```diff
@@ -0,0 +1,11 @@
+import { AppTheme } from '../enums'
+import { IAppSettings } from '../interfaces'
+
+export const DEFAULT_APP_SETTINGS: IAppSettings = {
+    deepLinking: true,
```

Comment:
Shouldn't we keep deepLinking `false` and request deepLink permission first?

Reply:
I actually don't even know if it's used any more. The setting doesn't make sense because it doesn't do anything with registering a url schema with the OS. So any permissions about what protocol is registered with the OS is managed by the OS. We should in future look at allowing the user to register and deregister through the app. But false doesn't do anything (at least previously) because the url schema is still registered and instead of following the link it just takes you to settings?

Reply:
Hmmm, I guess I'm not too familiar with how the deep links work under the hood because the part about url schemas and the OS is going over my head

...but this changed DEFAULT_APP_SETTINGS does change the default value of the deepLink setting in newly created profiles (just checked)

Reply:
Let me revert that change and we can make it in a separate Pr to be clear what we are doing with that setting

Reply:
As for new profiles this change will only affect fresh installs, or you can delete the settings object from application data to test